docs/Common/Structs/index.md

```diff
@@ -1179,3 +1182,40 @@ There are no defined parameters for this struct
 |:---|:---|:--------|:---------|:----------|
 |location|Common.Grid|true|||
 |state|Common.WindowState|true|||
+
+### DoorStatus
+
+|Name|Type|Mandatory|Additional|Description|
+|:---|:---|:--------|:---------|:----------|
+|location|Common.Grid|true||Describes the status of a location of a door.|
+|status|[Common.DoorStatusType](../enums/#doorstatustype)|true||Describes the status of a door.|
+
+### GateStatus
+
+|Name|Type|Mandatory|Additional|Description|
+|:---|:---|:--------|:---------|:----------|
+|location|Common.Grid|true||Describes the status of location of trunk/hood/etc.|
+|status|[Common.DoorStatusType](../enums/#doorstatustype)|true||Describes the status of trunk/hood/etc.|
+
+### RoofStatus
+
+|Name|Type|Mandatory|Additional|Description|
+|:---|:---|:--------|:---------|:----------|
+|location|Common.Grid|true||Describes the status of a parameter of roof, convertible roof, sunroof/moonroof etc.<br>If roof is open (AJAR), state will determine percentage of roof open.|
```

Comment:
Please add reference links for Grid, DoorStatusType & WindowState

Reply:
the same as above, `Grid`, `WindowState` are structs and are not worth adding links in Structs index.
`DoorStatusType` is enum, the link is there (line 1205)

`|status|[Common.DoorStatusType](../enums/#doorstatustype)|true||Describes the status of a parameter of roof, convertible roof, sunroof/moonroof etc.<br>If roof is open (AJAR), state will determine percentage of roof open.|`

Reply:
Same [comment as above](https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/pull/232#discussion_r540980462)

Reply:
@atiwari9, please see the links to enums updated per your request in 82d254e

Reply:
@KhrystynaDubovyk  - The case looks good and it seems it links should work. But i believe it is still expecting: `Common/Enums/index.md#doorstatustype` instead of `Common/Enums/#doorstatustype` . I think this should by default point to `doorStatusType` in `index.md`, but somehow on my machine it is not doing that. Is that working for you?

Reply:
well, what I did initially is - I simply wrote the relative links to enums in **Structs.index** in the same fashion the PM does in its PRs (e.g. see https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/pull/179/files)

These links are written the following way
`[Common.*EnumName*](../enums/#*enumname*)` 
and I expect the links work on [the Portal](https://smartdevicelink.com/en/docs/hmi/master/common/structs/) once the PR is merged. 

I suspect if I write the link this way 
`Common/Enums/index.md#doorstatustype`
it might work for the PR, but wouldn't work correctly for the portal.

Reply:
Ok.